adafruit_bno055.py

```diff
@@ -139,37 +145,41 @@
 
 
 class _ScaledReadOnlyStruct(Struct):  # pylint: disable=too-few-public-methods
-    def __init__(self, register_address, struct_format, scale):
+    def __init__(self, register_address: int, struct_format: str, scale: float) -> None:
         super().__init__(register_address, struct_format)
         self.scale = scale
 
-    def __get__(self, obj, objtype=None):
+    def __get__(self, obj, objtype=None) -> Tuple[float, float, float]:
         result = super().__get__(obj, objtype)
         return tuple(self.scale * v for v in result)
 
-    def __set__(self, obj, value):
+    def __set__(self, obj, value) -> None:
```

Comment:
should the `obj` and `value` here have types? In the __set__ function a little below here it has `int` and `str` for their types, would it be the same for this one?

Reply:
Must of missed them? In ny until the weekend with just my phone, so nothing in this till then